### DISCUSSION TITLE
Panel settings: New option to fill up the grid vertically

### DISCUSSION THREAD

**Discussion by ggregoire** [grafana/grafana#38129] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

An option per panel that would make the panels automatically fill up the grid vertically.

**Why is this needed**:

Dashboard looks fine on a laptop (screenshot 1) but half empty on an external monitor or a TV (screenshot 2).

<img width="1440" alt="Screen Shot 2021-07-14 at 8 30 28 PM" src="https://user-images.githubusercontent.com/591804/125714322-6f26c087-7140-4b47-a5c2-8cfaa8ce9ca5.png">

<img width="1919" alt="Screen Shot 2021-07-14 at 8 31 17 PM" src="https://user-images.githubusercontent.com/591804/125714338-c14d42a6-7589-4fd0-bc36-67de1c2e4000.png">

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
